go.mod

```diff
@@ -50,8 +50,4 @@ require (
 
 replace github.com/gogo/protobuf => github.com/ElrondNetwork/protobuf v1.3.2
 
-replace github.com/ElrondNetwork/arwen-wasm-vm/v1_2 v1.2.42-0.20220729115258-b9f2fb2f6568 => github.com/ElrondNetwork/arwen-wasm-vm v1.2.42-0.20220729115258-b9f2fb2f6568
-
-replace github.com/ElrondNetwork/arwen-wasm-vm/v1_3 v1.3.42-0.20220729115131-85ecca868e90 => github.com/ElrondNetwork/arwen-wasm-vm v1.3.42-0.20220729115131-85ecca868e90
-
```

Comment:
do we need the following replace for the v1.4 version?

Reply:
This PR will eventually obviate the need for the final `replace`. Work in progress,

Reply:
Done.